code/__HELPERS/traits.dm

```diff
@@ -216,9 +222,30 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define CULT_EYES "cult_eyes"
 #define DOGGO_SPACESUIT "doggo_spacesuit"
 #define FLOORCLUWNE "floorcluwne"
+#define LOCKDOWN_TRAIT "lockdown"
+#define STAT_TRAIT "stat_trait"
+#define TRANSFORMING_TRAIT "transforming"
+#define BUCKLING_TRAIT "buckled"
 
 //quirk traits
 #define TRAIT_ALCOHOL_TOLERANCE	"alcohol_tolerance"
 
 //traits that should be properly converted to genetic mutations one day
 #define TRAIT_LASEREYES "laser_eyes"
+
+//status effec traits
+/// Forces the user to stay unconscious.
+#define TRAIT_KNOCKEDOUT "knockedout"
+/// Prevents voluntary movement.
+#define TRAIT_IMMOBILIZED "immobilized"
+/// Prevents voluntary standing or staying up on its own.
+#define TRAIT_FLOORED "floored"
+/// Prevents usage of manipulation appendages (picking, holding or using items, manipulating storage).
+#define TRAIT_HANDS_BLOCKED "handsblocked"
+/// Inability to access UI hud elements.
+#define TRAIT_UI_BLOCKED "uiblocked"
+/// Inability to pull things.
+#define TRAIT_PULL_BLOCKED "pullblocked"
```

Comment:
I think this one might more clear if it were `TRAIT_CANNOT_PULL`. Right now, to me it reads like the thing with the trait cannot be pulled.